ivy/functional/frontends/numpy/broadcast/methods.py

```diff
@@ -0,0 +1,56 @@
+# local
+import ivy
+from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back
+
+
+class broadcast:
+    @to_ivy_arrays_and_back
+    def __init__(self, *args):
+        data = ivy.broadcast_arrays(*map(ivy.array, args))
+        self.__shape = data[0].shape
```

Comment:
Can we use one underscore here 
something like `self._shape` as this one doesn't look good

Reply:
Hi @hirwa-nshuti, updated it.